File: src/main/java/com/likelionknu/applyserver/application/data/dto/response/ApplicationInfoResponseDto.java
package com.likelionknu.applyserver.application.data.dto.response;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ApplicationInfoResponseDto {
    private String name;
    private String email;
    private String depart;

    @JsonProperty("student_id")
    private String studentId;

    private Integer grade;

    @JsonProperty("student_status")
    private String studentStatus;

    private String status;

    @JsonProperty("submitted_at")
    private LocalDateTime submittedAt;
    private String phone;
    private List<ApplicationAnswerResponseDto> answers;
}

File: src/main/kotlin/com/likelionknu/applyserver/application/data/dto/response/ApplicationInfoResponseDto.kt
package com.likelionknu.applyserver.application.data.dto.response

import com.fasterxml.jackson.annotation.JsonProperty
import java.time.LocalDateTime

data class ApplicationInfoResponseDto (
    val name: String,
    val email: String,
    val depart: String? = null,

    @field:JsonProperty("student_id")
    val studentId: String? = null,

    val grade: Int? = null,

    @field:JsonProperty("student_status")
    val studentStatus: String? = null,

    val status: String,

    @field:JsonProperty("submitted_at")
    val submittedAt: LocalDateTime,

    val phone: String? = null,
    val answers: List<ApplicationAnswerResponseDto>
)